### PR TITLE
fix design issues on repo summary page

### PR DESCRIPTION
**Pull request by abhinavrastogi-harness**
After fixes:

Screenshot 2024-12-11 at 3 39 12 PM

Screenshot 2024-12-11 at 3 39 03 PM

### CODE REVIEW

File: packages/ui/locales/en/views.json [hunk 1]
```diff
@@ -37,10 +37,10 @@
     "create-repository": "Create repository",
     "private": "Private",
     "public": "Public",
-    "add-file": "Add file",
-    "createNewFile": "+ Create New File",
     "readme": "README.md",
-    "openPullReq": "Open pull requests"
+    "openPullReq": "Open pull requests",
+    "add-file": "Add file",
```
Comment (cjlee01) [inline packages/ui/locales/en/views.json:42]: shud this be camelcase?

Comment (cjlee01) [reply packages/ui/locales/en/views.json:42]: ```addFile```

File: packages/ui/locales/es/views.json [hunk 2]
```diff
@@ -37,10 +37,10 @@
     "create-repository": "",
     "private": "Private",
     "public": "Public",
-    "add-file": "Add file",
-    "createNewFile": "+ Create New File",
     "readme": "README.md",
-    "openPullReq": "Open pull requests"
+    "openPullReq": "Open pull requests",
+    "add-file": "Add file",
```
Comment (cjlee01) [inline packages/ui/locales/es/views.json:42]: same thing here

**Review by sans-harness — APPROVED** [review]

File: packages/ui/src/views/repo/repo-summary/repo-summary.tsx [hunk 3]
```diff
@@ -125,24 +125,15 @@ export function RepoSummaryView({
...
-                        }}
-                      >
-                        {t('views:repos.createNewFile', '+ Create New File')}
-                      </DropdownMenuItem>
-                    </DropdownMenuContent>
-                  </DropdownMenu>
+                  <Button variant={'secondary'} asChild>
+                    <Link to={`/${spaceId}/repos/${repoId}/code/new/${gitRef || selectedBranchTag?.name || ''}/~/`}>
+                      Add File
+                    </Link>
```
Comment (sans-harness) [inline packages/ui/src/views/repo/repo-summary/repo-summary.tsx:131]: can we get relative routing here?

File: packages/ui/src/views/repo/repo-summary/repo-summary.tsx [hunk 4]
```diff
@@ -125,24 +125,15 @@ export function RepoSummaryView({
...
-                          navigate(`/${spaceId}/repos/${repoId}/code/new/${gitRef || selectedBranchTag?.name || ''}/~/`)
-                        }}
-                      >
-                        {t('views:repos.createNewFile', '+ Create New File')}
-                      </DropdownMenuItem>
-                    </DropdownMenuContent>
-                  </DropdownMenu>
+                  <Button variant={'secondary'} asChild>
+                    <Link to={`/${spaceId}/repos/${repoId}/code/new/${gitRef || selectedBranchTag?.name || ''}/~/`}>
+                      Add File
```
Comment (cjlee01) [inline packages/ui/src/views/repo/repo-summary/repo-summary.tsx:130]: do we use usetranslation t() here?

Comment (cjlee01) [reply packages/ui/src/views/repo/repo-summary/repo-summary.tsx:130]: ```{t('views:repos.addFile', 'Add File')}```

File: packages/ui/src/views/repo/repo-summary/repo-summary.tsx [hunk 5]
```diff
@@ -125,24 +125,15 @@ export function RepoSummaryView({
...
-                        key={'create-file'}
-                        onClick={() => {
-                          navigate(`/${spaceId}/repos/${repoId}/code/new/${gitRef || selectedBranchTag?.name || ''}/~/`)
-                        }}
-                      >
-                        {t('views:repos.createNewFile', '+ Create New File')}
-                      </DropdownMenuItem>
-                    </DropdownMenuContent>
-                  </DropdownMenu>
+                  <Button variant={'secondary'} asChild>
```
Comment (knagurski) [inline packages/ui/src/views/repo/repo-summary/repo-summary.tsx:128]: Nitpick
```suggestion
                  <Button variant="secondary" asChild>
```

**Pull request merged**